### ISSUE TITLE
8545ebb743fb8787d406471faede781d59b1cce6 creates modpack, which has not been approved

### ISSUE THREAD

**Issue by sofar** [Mynetest/throwing#18]
Please revert this commit, or state your arguments why you should be allowed to break this mod into a modpack against the rules.

**Comment by paly2** [comment]
Sorry I really didn't think it would be so controversial. I understand the importance of the rules stated in the manifest, re-created a throwing_arrows repository and transferred you its ownership so it can be added back to minetest-mods (I'd prefer to wait for the repository to be added back before pushing the revert commits).

Still, the arguments that made me turn the mod into a modpack are mainly that I felt over-killing to have to download two repositories for a single feature (bows and arrows), but I understand it's just a personal judgment matter. Throwing shows no functionality unless used with another mod (okay it's not necessarily throwing_arrows) and throwing_arrows cannot be used unless throwing is installed, so it seemed quite natural to me that it would be simpler, cleaner and clearer for the players who use the mods if they would be grouped. It's really nothing like a modpack grouping unrelated mods, it is more like mesecons: you can use mesecons alone, you can also use mesecons with third-party mods, but you are much more likely to want the other components of the modpack as well.

So, to sum up: I'd like to make it a modpack for these reasons, but if you believe it's a bad idea I will not argue anymore against it. I apologize for the mess it seems to have caused, apologize for not having read the manifest thoroughly enough before taking the decision, and will push revert commits as soon as throwing_arrows is transferred back to minetest-mods.

**Comment by sofar** [comment]
hey, thanks for taking the time to discuss.

I'm not opposed to modpacks BTW, but you have lots of alternative ways to distribute them:

- you can tag releases and upload modpack zips to github projects!
- add them to forum posts
- etc

so, you can still solve the end user problem while keeping components separate so that people can experiment with the API and come up with other cool uses without forcing everyone to use the arrows that are part of the mod. The development model has worked well and created more usable components, which is crucial for open source. Nobody can maintain one project with hundreds of parts. Many people can maintain one project themselves and make sure others don't break. This is why mesecons, technic etc' large collection is such a problem to maintain. I hate to see that happen to other mods.

**Comment by paly2** [comment]
> you can tag releases and upload modpack zips to github projects!

That sounds like a great idea :-) I'm waiting for you to transfer the new [throwing_arrows repository](https://github.com/paly2/throwing_arrows) to minetest-mods -- if you agree to do that, of course -- and will push revert commits so this repo becomes back only the API

**Comment by sofar** [comment]
I pulled back throwing_arrows from the last checkout I had. If I missed any commits, just push them back.

**Comment by paly2** [comment]
Thanks, I've pushed the revert commits :-)
